### PR TITLE
[add]: readme, code-of-conduct & contributing guidelines

### PR DESCRIPTION
**Pull request by rahuldkjain**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+```bash
+npm run dev
+```
+
+## 📃 Documentation
+Please refer the [documentation](https://analytics.cool.bio/docs) for more details.
+
+## 👩‍💻 Tech
+- React
+- Recharts
```
Comment (subhendukundu) [inline README.md:60]: Would you mind adding vitedge.js.org, cloudflare, hasura.io?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+cd cool-bio-analytics
+```
+**Step 2**: Install the dependencies
+```bash
+npm install
+```
+
+**Step 3**: Run it locally
+```bash
+npm run dev
```
Comment (subhendukundu) [inline README.md:52]: Would be great if you switch it to `npm run dev:ssr`

**Review by subhendukundu — COMMENTED** [review]
Looking great. Just two small changes.

File: README.md (hunk 1 above)
Comment (rahuldkjain) [reply README.md:60]: Sure

File: README.md (hunk 2 above)
Comment (rahuldkjain) [reply README.md:52]: Got it

**Pull request merged**